datastore/elastic_search/query.py

```diff
@@ -282,17 +283,27 @@ def full_text_query(connection, index_name, doc_type, entity_name, sentence, fuz
          u'mumbai': u'mumbai',
          u'pune': u'pune'}
     """
-    data = _generate_es_search_dictionary(entity_name, sentence, fuzziness_threshold,
-                                          language_script=search_language_script)
-    kwargs = dict(kwargs, body=data, doc_type=doc_type, size=constants.ELASTICSEARCH_SEARCH_SIZE, index=index_name)
+    data = [
+        _generate_es_search_dictionary(entity_name, sentence_, fuzziness_threshold,
+                                       language_script=search_language_script) for sentence_ in sentence]
+    # kwargs = dict(kwargs, body=data, doc_type=doc_type, size=constants.ELASTICSEARCH_SEARCH_SIZE, index=index_name)
+    query_data = []
+    index_for_each_query = {}
+    for query in data:
+        query_data.extend([index_for_each_query, query])
+    request = ''
+    for each in query_data:
+        request += '%s \n' % json.dumps(each)
```

Comment:
For lines 286 - 296 it should be possible to  simplify logic
```python
index_ = {'index': index_name, 'type': doc_type}'
data = []
for sentence in sentences:
    query = _generate_es_search_dictionary(entity_name, sentence_, fuzziness_threshold, language_script=search_language_script)
    data.extend([json.dumps(index_), json.dumps(query)])
data = '\n'.join(data)
```

Reply:
Done